attributes/default.rb

```diff
@@ -18,5 +18,5 @@
 # limitations under the License.
 #
 
-default['gluster']['version'] = '3.4'
+default['gluster']['version'] = '3.6'
```

Comment:
`3.7.6` is the newest version.  Should we just bump it all the way up since this is a major rewrite anyway?